modules/samples-operator-configuration.adoc

```diff
@@ -9,21 +9,19 @@
 
 The samples resource offers the following configuration fields:
 
-* *ManagementState:*
+* *Management State:*
 ** *Managed.* The operator updates the samples as the configuration dictates.
-** *Unmanaged.* The operator ignores updates to the samples resource object and
-any image streams or templates in the OpenShift namespace.
+** *Unmanaged.* The operator ignores updates to its configuration resource object
+and any image streams or templates in the OpenShift namespace.
```

Comment:
All 'image streams' phrases in this doc https://docs.openshift.com/container-platform/4.0/openshift_images/configuring-samples-operator.html#samples-operator-configuration-configuring-samples-operator should be updated to 'imagestreams'